lease/lessor.go

```diff
@@ -68,9 +75,15 @@ type lessor struct {
 	idgen *idutil.Generator
 }
 
-func NewLessor(lessorID uint8, b backend.Backend, dr DeleteableRange) *lessor {
+func NewLessor(lessorID uint8, b backend.Backend, dr DeleteableRange) Lessor {
```

Comment:
`func NewLessor(...) Lessor { return newLessor(...) }`
`func newLessor(...) *lessor { ... }`

To avoid all the casts in `lessor_test.go`